whatwg.org/.htaccess

```diff
@@ -14,8 +14,8 @@ AddType image/gif .gif
 # Indexing Options
 IndexOptions DescriptionWidth=* NameWidth=* FancyIndexing FoldersFirst ScanHTMLTitles
 # also add SupressHTMLPreamble once we use HeaderName and some fancy CGI
-IndexIgnore .htaccess *~ .#* #*# CVS .svn favicon.ico favicon.gif robots.txt
```

Comment:
robots.txt certainly exists... I guess not in any directories without a index page?

Reply:
Exactly, there just one in the root. I'll update the commit message.